plugins/microsoft_log_analytics/icon_microsoft_log_analytics/util/endpoints.py

```diff
@@ -14,3 +14,15 @@ class Endpoint:
 
     SEND_LOG_DATA = "https://{}.ods.opinsights.azure.com/api/logs?api-version={}"
     GET_LOG_DATA = RESOURCE_LOG_API + "/{}/workspaces/{}/query"
+    GET_SAVED_SEARCH = (
+        RESOURCE_MANAGEMENT
+        + "/subscriptions/{}/resourcegroups/{}/providers/Microsoft.OperationalInsights/workspaces/{}/savedSearches/{}?api-version={}"
+    )
+    DELETE_SAVED_SEARCH = GET_SAVED_SEARCH
+    CREATE_OR_UPDATE_SAVED_SEARCH = GET_SAVED_SEARCH
+    LIST_ALL_SEARCHES = (
+        RESOURCE_MANAGEMENT
+        + "/subscriptions/{}/resourcegroups/{}/providers/Microsoft.OperationalInsights/workspaces/{}/savedSearches?api-version={}"
+    )
+
+    API_VERSION_SAVED_SEARCH = "2020-08-01"
```

Comment:
Is this indeed supposed to be a hardcoded date? Or was this for testing?

Reply:
It is hardcoded in order for easy change in the future (if needed)